c7n/resources/health.py

```diff
@@ -78,10 +78,24 @@ def resources(self, query=None):
             query['filter'] = q
         return super(HealthEvents, self).resources(query=query)
 
+    def augment(self, resources):
+        client = local_session(self.session_factory).client('health')
+        for r in resources:
+            r['eventDescription'] = client.describe_event_details(
+                eventArns=[r['arn']])['successfulSet'][0]['eventDescription']
+            if r['eventTypeCategory'].encode('utf8') != 'accountNotification':
```

Comment:
rather than encoding the eventTypeCategory you can just do a u'accountNotification'